lib/packwerk/checking_references/checkers/README.md

```diff
@@ -0,0 +1,19 @@
+# Packwerk::Checkers
+
+- Checker implements `invalid_reference?(reference)`, which takes a `Packwerk::Reference` and returns Boolean.
```

Comment:
Nitpick: This is accurate, but it duplicates the interface definition in `Packwerk::Checker`, so I think in the final docs we should just point out that the interface exists instead of putting the details here